astropy/uncertainty/core.py

```diff
@@ -201,7 +201,7 @@ def pdf_percentiles(self, percentile, **kwargs):
         Parameters
         ----------
         percentile : float or array of float or `~astropy.units.Quantity`
-            The desired  precentiles of the distribution (i.e., on [0,100]).
+            The desired  percentiles of the distribution (i.e., on [0,100]).
```

Comment:
Might as well fix the spacing here while we're at it.
```suggestion
            The desired percentiles of the distribution (i.e., on [0,100]).
```